src/OMSimulatorLib/System.cpp

```diff
@@ -1628,6 +1628,16 @@ oms_status_enu_t oms::System::delete_(const oms::ComRef& cref)
     auto component = components.find(front);
     if (component != components.end())
     {
+      if (Flags::ExportParametersInline())
+      {
+        // delete startValues associated with the Connector
+        component->second->deleteStartValue(tail);
+      }
+      else
+      {
+        // delete startValues associated with the Connector from ssv file
+        component->second->deleteStartValue(cref);
+      }
```

Comment:
Please move this to `oms::Component::deleteConnector`

Reply:
moved the code to `oms::Component::deleteConnector` see https://github.com/OpenModelica/OMSimulator/pull/758/commits/b669e30afeea6d53ecfb6f68d9e9e71474db8dfc